docs/framework-support-libraries.md

```diff
@@ -136,15 +138,45 @@ and define the provider:
  ```
 
 
+### LuigiMockModule
+
+In the normal Luigi workflow messages coming from Luigi Client to Luigi Core are processed on the Core and a proper response is sent back. However, in many systems where testing of micro frontends standalone is a necessity, the absence of Luigi Core to send back needed responses to Client micro frontends becomes a case of high coupling. To remove this coupling we introduce **LuigiMockModule** for **Angular** applications. This module is attached to the start of your application where it intercepts all the Client calls and sends a mocked Core response back. This enables users to test their micro frontends standalone without depending on the Core. 
+To use **LuigiMockModule**, simply add it to the list of imports of your applications entry point. A good practice is to include it in the main testing module of your application as given in the example below:
+
+ ```javascript
+import {LuigiMockModule} from '@luigi-project/client-support-angular';
+
+/**
+ * This module is used to run the application for e2e tests.
+ */
+@NgModule({
+  imports: [
+    AppModule,
+    LuigiMockModule,
+  ],
+  bootstrap: [AppComponent]
+})
+export class AppTestingModule {}
+
+ ```
+
+To make mocking of Luigi Core easier, you can use a range of utility functions and assertions. Our lightweight [Luigi Testing Utilities](https://docs.luigi-project.io/docs/framework-support-libraries/?section=luigi-testing-utilities) library provides the necessary basic utility functions needed for your application. 
+
+
+
 ## Luigi Testing Utilities
 
-Luigi Testing Utilities is a set of auxiliary functions used to enhance the user experience while testing Luigi based microfrontends. The functions abstract away Luigi specific logic from the tester so that it is easier for them to test. 
+Luigi Testing Utilities is a set of auxiliary functions used to enhance the user experience while testing Luigi based micro frontends. The functions abstract away Luigi specific logic from the tester so that it is easier for them to mock and assert Luigi functionality. 
```

Comment:
```suggestion
The Luigi Testing Utilities are a set of auxiliary functions used to enhance the user experience while testing Luigi-based micro frontends. The functions abstract away Luigi-specific logic from the tester so that it is easier for them to mock and assert Luigi functionality. 
```